### PR TITLE
Follow-up to Add in extra warnings for Cassandra and its disk size(PR#1877)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1877

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @tpoitras @bfallonf PTAL

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -168,15 +168,15 @@ $ oc process -f metrics-deployer.yaml -v \
 
 [WARNING]
 ====
-The size requirement of the Cassandra storage is dependent on the cluster size. It is
-the admin's responsibility to make sure that the size requires are sufficent for their setup and
-to monitor usage to make sure that the disk does not become full.
+The size requirement of the Cassandra storage is dependent on the cluster size.
+It is the administrator's responsibility to ensure that the size requirements
+are sufficient for their setup and to monitor usage to ensure that the disk does
+not become full.
```
Comment (adellape) [inline install_config/cluster_metrics.adoc]: @ahardin-rh Suggest treating this like the WARNING admonition later, with just the "Data loss..." sentence being in the box.

Comment (adellape) [inline install_config/cluster_metrics.adoc]: And un-bolded.

**Comment by adellape** [comment]
@ahardin-rh One comment inline, otherwise LGTM.

**Comment by ahardin-rh** [comment]
@adellape Thanks! Fixed!

**Comment by ahardin-rh** [comment]
Rev history lives in https://github.com/openshift/openshift-docs/pull/1877

**Pull request merged**